Zend/Optimizer/zend_optimizer.c

```diff
@@ -620,7 +620,7 @@ bool zend_optimizer_update_op2_const(zend_op_array *op_array,
 
 bool zend_optimizer_replace_by_const(zend_op_array *op_array,
                                     zend_op       *opline,
-                                    zend_uchar     type,
+                                    uint8_t     type,
```

Comment:
```suggestion
                                    uint8_t        type,
```

Reply:
Oh, this weird coding style. I didnt pay attention to that because the existing code is inconsistent already. Most of the code base does no line breaking at all, and these three break after each parameter.
But in this particular function, the indentation of the follow-up parameter line is one off to the left.